### ISSUE TITLE
Unable to use Downloader page

### ISSUE THREAD

**Issue by ArTBoL** [rimpy-custom/RimPy#84]
Hi
i'm on Kubuntu and Rimpy [1.2.6.25]
the downloader page dosent work, only white page.

> ~/Games/mods/RimPy_Linux$ ./RimPy.sh 
Sandboxing disabled by user.
Ошибка открытия URL: https://api.github.com/repos/rimpy-custom/RimPy/releases
Ошибка открытия URL: https://content-system.gog.com/products/1094900565/os/windows/builds?generation=2
Ошибка открытия URL: https://rimworldwiki.com/wiki/Version/Current_version
0.6777462860000014
[9955:9971:0821/131926.578032:ERROR:nss_util.cc(166)] Error initializing NSS with a persistent database (sql:/home/artbol/.pki/nssdb): libsoftokn3.so: cannot open shared object file: No such file or directory
[9955:9971:0821/131926.578080:ERROR:nss_util.cc(124)] Error initializing NSS without a persistent database: NSS error code: -5925
[9955:9971:0821/131926.578088:FATAL:nss_util.cc(126)] nss_error=-5925, os_error=0
[9890:9890:0821/131926.677581:ERROR:network_service_instance_impl.cc(262)] Network service crashed, restarting service.
js: Uncaught TypeError: Cannot read property 'lastChild' of undefined
js: Uncaught TypeError: Cannot read property 'parentElement' of null

[rimpy.log](https://github.com/rimpy-custom/RimPy/files/9388964/rimpy.log)

**Comment by Pangaea666** [comment]
I have the same problem. The Downloader page is white, and I get almost exactly the same error message when launching RimPy via Terminal. I'm on Linux Mint 20.3 if that matters. 

```
Gtk-Message: 03:09:49.838: Failed to load module "xapp-gtk3-module"
Sandboxing disabled by user.
Ошибка открытия URL: https://api.github.com/repos/rimpy-custom/RimPy/releases
Ошибка открытия URL: https://content-system.gog.com/products/1094900565/os/windows/builds?generation=2
Ошибка открытия URL: https://rimworldwiki.com/wiki/Version/Current_version
0.7745191430003615
[35823:35832:0915/030950.697989:ERROR:nss_util.cc(166)] Error initializing NSS with a persistent database (sql:/home/pangaea/.pki/nssdb): libsoftokn3.so: cannot open shared object file: No such file or directory
[35823:35832:0915/030950.698032:ERROR:nss_util.cc(124)] Error initializing NSS without a persistent database: NSS error code: -5925
[35823:35832:0915/030950.698045:FATAL:nss_util.cc(126)] nss_error=-5925, os_error=0
[35788:35788:0915/030950.799354:ERROR:network_service_instance_impl.cc(262)] Network service crashed, restarting service.
js: Uncaught TypeError: Cannot read property 'lastChild' of undefined
js: Uncaught TypeError: Cannot read property 'parentElement' of null
```
![RimPy problem](https://user-images.githubusercontent.com/2002335/190290363-0cb53bc9-c8ef-4852-8d69-fd7e7f515315.png)
[rimpy.log](https://github.com/rimpy-custom/RimPy/files/9570971/rimpy.log)

**Comment by NBZion** [comment]
to fix this problem you have to install libnss or nss

**Comment by Pangaea666** [comment]
It is already installed, though I'm not 100% sure on the version number. In any case, I'm seeing people report (here and on Discord) the exact same issue with bleeding edge releases, so it's possible Rimpy requires one very specific version for some reason.

**Comment by sweetkid169** [comment]
#91 I posted about something similar to this, except it is SteamCMD related for me.

**Comment by rimpy-custom** [comment]
Closing for now. Reopen if issue wasn't fixed with latest Linux build 1.2.6.28.
